src/Nox.Solution/Models/Application/Integrations/Schedule/IntegrationSchedule.cs

```diff
@@ -20,5 +20,5 @@ public class IntegrationSchedule
 
     [Title("Check the ETL source for updates at appliation startup.")]
     [Description("Specify here whether the ETL source is checked for updates when the application starts.")]
-    public bool? RunOnStartup { get; internal set; }
+    public bool? RunOnStartup { get; internal set; } = false;
```

Comment:
can we remove the nullable since is not required and the default is false? so we can avoid null checks in code?
```suggestion
    public bool RunOnStartup { get; internal set; } = false;
```